code/modules/clothing/suits/wiz_robe.dm

```diff
@@ -218,14 +216,5 @@
 	desc = "A powerful rune that will increase the number of hits a suit of battlemage armour can take before failing.."
 	icon = 'icons/effects/effects.dmi'
 	icon_state = "electricity2"
-
-/obj/item/wizard_armour_charge/afterattack(obj/item/clothing/suit/space/hardsuit/shielded/wizard/W, mob/user, proximity)
-	. = ..()
-	if(!proximity)
-		return
-	if(!istype(W))
-		to_chat(user, "<span class='warning'>The rune can only be used on battlemage armour!</span>")
-		return
-	W.current_charges += 8
-	to_chat(user, "<span class='notice'>You charge \the [W]. It can now absorb [W.current_charges] hits.</span>")
-	qdel(src)
+	/// How many charges get restored
+	var/restored_charges = 8
```

Comment:
This is used where?

Reply:
```/datum/component/shielded/proc/check_recharge_rune(datum/source, obj/item/wizard_armour_charge/recharge_rune, mob/living/user)
	SIGNAL_HANDLER

	if(!istype(recharge_rune))
		return
	. = COMPONENT_NO_AFTERATTACK
	if(!istype(parent, /obj/item/clothing/suit/space/hardsuit/shielded/wizard))
		to_chat(user, "<span class='warning'>The rune can only be used on battlemage armour!</span>")
		return

	current_charges += recharge_rune.restored_charges
	to_chat(user, "<span class='notice'>You charge \the [parent]. It can now absorb [current_charges] hits.</span>")```
there

Reply:
Added a comment pointing to the use